opencti-platform/opencti-front/src/private/components/workspaces/WorkspaceCreation.jsx

```diff
@@ -135,6 +134,7 @@ const WorkspaceCreation = ({ paginationOptions, type }) => {
       >
         {t_i18n('Import dashboard')}
       </Button>
+      <CreateEntityControlledDial entityType='Dashboard' {...props} />
```

Comment:
we keep the same order everywhere : from left to right = tertiary > secondary > primary